src/Controller/ExportController.php

```diff
@@ -129,11 +135,14 @@ private function export( Request $request, CreationLog $creationLog, LoggerInter
 	 * @param string $lang A language code.
 	 * @return string
 	 */
-	private function getFont( Request $request, $lang ): ?string {
+	private function getFont( Request $request, $lang, FontProvider $fontProvider ): ?string {
 		// Default font for non-latin languages.
-		$font = $request->get( 'fonts', '' );
+		$font = $fontProvider->resolveName( $request->get( 'fonts', '' ) );
```

Comment:
Maybe I'm missing something but `$fontProvider->getOne()` calls `resolveName()` internally so this call is unnecessary. If that's the case, `FontProvider::resolveName()` can be set as private. Does that makes sense?

Reply:
Good idea about calling it inside `getCss()`. I'm afraid I still don't understand why it needs to be called in the controller and it looks to me that it should be an implementation detail of `FontProvider` but I might be missing something. LGTM otherwise